.chloggen/coralogix-exporter.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: coralogixexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "allow setting application name from cx.application.name and cx.subsystem.name attributes"
```

We should specify that these are resource attributes
```suggestion
note: "Allow setting application name from `cx.application.name` and `cx.subsystem.name` resource attributes"
```